src/emulator/cpu.ts

```diff
@@ -609,6 +625,15 @@ export class Cpu {
         return instruction.cycles;
     }
 
+    private opLds(instruction: Instruction): number {
```

Comment:
Hm, das is identisch mit `opLd`, oder? Also, ich würde entweder `Operation.lds` zugunsten  `Operation.ld` und den neuen Adressierungsmodus abschaffen, oder den Adressierungsmodus zugunsten von `lds` mit `AdressingMode.imm8` abschaffen 😏

Reply:
LD HL, SP+8 setzt leider die Flags anders als die anderen LD Instruktionen. Deswegen ist das eine eigene Operation.

Reply:
Hm? Ich sehe da nichts von Flags, und auch nirgendwo einen Hinweis im Netz.

```
private opLds(instruction: Instruction): number {
        this.clock.increment(instruction.cycles);

         this.setArg1(instruction, this.getArg2(instruction));

         this.state.p = (this.state.p + instruction.len) & 0xffff;
         return instruction.cycles;
     }
````

Reply:
Ups, da hab ich wohl etwas vergessen, oder ist im Rebase auf deine Änderungen mit den OpCodes verloren gegangen.... :D

Reply:
👻

Reply:
Flags sind jetzt drin. :)